update-tags.mjs

```diff
@@ -0,0 +1,56 @@
+import { readFile, writeFile } from 'fs/promises'
+import { glob as baseGlob } from 'glob'
+import packageJSON from './package.json' assert { type: 'json' }
+import * as prettier from 'prettier'
+
+const tagPath = `/v${packageJSON.version}/`
+
+async function glob(path) {
+  return await baseGlob(path, {
+    ignore: ['node_modules/**'],
+  })
+}
+
+async function fileUpdates(fileSet, key, oldValue, newValue) {
+  await Promise.all(
+    fileSet.map(async (fileItem) => {
+      let file = JSON.parse(
+        await readFile(fileItem, {
+          encoding: 'utf-8',
+        })
+      )
+      file[key] = file[key].replace(
+        `https://gcn.nasa.gov/schema/${oldValue}`,
+        `https://gcn.nasa.gov/schema/${newValue}`
+      )
+      await writeFile(fileItem, JSON.stringify(file))
+    })
+  )
+}
+
+async function loadAndUpdateFiles(oldValue, newValue) {
+  const schemaFilenames = await glob('**/*.schema.json')
+  await fileUpdates(schemaFilenames, '$id', oldValue, newValue)
+
+  const exampleFilenames = await glob('**/*.example.json')
+  await fileUpdates(exampleFilenames, '$schema', oldValue, newValue)
+}
+
+if (process.argv.includes('--reset')) {
+  await loadAndUpdateFiles(tagPath, '/main/')
+} else {
+  await loadAndUpdateFiles('/main/', tagPath)
+}
+
+const files = await glob('**/*.json')
+for (const file of files) {
+  const text = await readFile(file, 'utf-8')
+  await writeFile(
+    file,
+    prettier.format(text, {
+      semi: false,
+      singleQuote: true,
+      parser: 'json',
+    })
+  )
```

Comment:
Run prettier inside `loadAndUpdateFiles` so that we don't read and write the same files multiple times.